packages/dx-react-grid/docs/guides/virtual-scrolling.md

```diff
@@ -38,6 +38,14 @@ If the Grid should have the same size as the container element, set the `Virtual
 
 .embedded-demo({ "path": "grid-virtual-scrolling/stretching-to-parent-element", "showThemeSelector": true })
 
+## Scroll to Row
+
+The virtual table can be scrolled programmatically, through the `scrollToRow` methods. To call these, you should use `VirtualTable` plugin's reference. The following demo demonstrates how scroll table to added row :
```

Comment:
`The following demo demonstrates how scroll table to added row` ->
`The following demo demonstrates how scroll table to new and added rows`

Reply:
Corrected